dev-util/bitcoin-tx/bitcoin-tx-25.1.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 at this point please

Reply:
No, we use the latest unless there's a reason not to. To not do that just makes life unnecessarily hard when it comes to formal deprecation and eventual banning. The only possible benefit is upgrade path and that ship has long sailed.